src/OpenRiaServices.Client/Framework/Internal/MetaType.cs

```diff
@@ -122,15 +122,15 @@ select attribute
             this.Type = type;
 
             _validationAttributes = new ReadOnlyCollection<ValidationAttribute>(this.Type.GetCustomAttributes(typeof(ValidationAttribute), true).OfType<ValidationAttribute>().ToArray());
-            _requiresValidation = _requiresValidation || _validationAttributes.Any();
+            _requiresValidation = _requiresValidation || _validationAttributes.Any() || type.GetInterface(typeof(IValidatableObject).FullName) != null;
```

Comment:
I suggest you use the general typeof(Interface).IsAssignableFrom(c.Type) which is used in many places across the codebase for consistency (and slightly better perf)

```suggestion
            _requiresValidation = _requiresValidation || _validationAttributes.Any() || typeof(IValidatableObject).IsAssignableFrom(type);
```